src/platformdirs/unix.py

```diff
@@ -125,6 +109,10 @@ def _first_item_as_path_if_multipath(self, directory: str) -> Path:
             directory = directory.split(os.pathsep)[0]
         return Path(directory)
 
+    @staticmethod
+    def _xdg_dir_or_fallback_with_expanded_user_dir(xdg_variable: str, fallback: str) -> str:
+        return os.environ.get(xdg_variable) or os.path.expanduser(fallback)
```

Comment:
Since a few default paths do not need expanding, wdyta just putting the appropriate one-liner in each function?

Reply:
One argument for the extracted function would be easier testability. Otherwise, I do not have a strong opinion about it.